op-monitorism/transaction_monitor/monitor_test.go

```
@@ -0,0 +1,237 @@
+)
+
+var (
+	// Anvil test accounts
+	watchedAddress   = common.HexToAddress("0xf39Fd6e51aad88F6F4ce6aB8827279cffFb92266")
+	allowedAddress   = common.HexToAddress("0x70997970C51812dc3A010C7d01b50e0d17dc79C8")
+	unauthorizedAddr = common.HexToAddress("0x3C44CdDdB6a900fa2b585dd299e03d12FA4293BC")
+	factoryAddress   = common.HexToAddress("0x90F79bf6EB2c4f870365E785982E1f101E93b906")
+
+	// Private keys
+	watchedKey, _ = crypto.HexToECDSA("ac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80")
+    neutralKey, _ = crypto.HexToECDSA("0x2a871d0798f97d79848a013d4936a73bf4cc922c825d33c1cf7073dff6d409c6")
```

golangci-lint (1.62.2)

var `neutralKey` is unused

(unused)